lib/lotus/entity.rb

```diff
@@ -171,6 +186,7 @@ def inherited(subclass)
 
     # Defines a generic, inefficient initializer, in case that the attributes
     # weren't explicitly defined with `.attributes=`.
+    # Clear dirty data if id is set.
```

Comment:
It only checks for the presence of ID, as keep track of where the model was initialized is not possible yet (from persisted level or by client)
So, if client set ID on init, it clear dirty data too.
Not best solution, but it's only way now to understand that model is persisted. Maybe you have ideas?